monix-execution/shared/src/main/scala/monix/execution/misc/Local.scala

```diff
@@ -148,14 +150,17 @@ object Local extends LocalCompanionDeprecated {
     def isolate(f: Tree): Tree =
       c.abort(c.macroApplication.pos, "Macro no longer implemented!")
 
-    def localLetCurrentIf(b: Tree)(f: Tree): Tree = {
+    def localLetCurrentIf(b: Tree, isolate: Tree)(f: Tree): Tree = {
```

Comment:
now it can be just `def localLetCurrentIf(b: Tree)`  because the behavior is the same for `b && isolate`
If we do it, we can also remove mima exclusion

Reply:
Maybe I am missing something about how macros work, but aren't the signatures meant to match, i.e.

```scala
private[monix] def bindCurrentIf[R](b: Boolean, isolate: Boolean)(f: => R): R
```

matches with

```scala
 def localLetCurrentIf(b: Tree, isolate: Tree)(f: Tree): Tree
```

I mean I need to check the `isolate` condition somehow?

Reply:
Or do you mean doing the isolate check inside of
```private[monix] def bindCurrentIf[R](b: Boolean, isolate: Boolean)(f: => R): R =```?

Reply:
I mean we could keep the old signature, that is:

`private[monix] def bindCurrentIf[R](b: Boolean)(f: => R): R`

And call it with `&&` in `Task`

Reply:
![DfKL_yPX4AAAaPG](https://user-images.githubusercontent.com/2337269/77835010-9daaf700-7149-11ea-932a-96800b308a43.jpg)

Fixed